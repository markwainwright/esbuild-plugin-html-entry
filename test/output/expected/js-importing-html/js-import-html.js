"use strict";
(() => {
  // test/input/pages/dead-end.html
  var dead_end_default = '<!doctype html>\n<html>\n  <head>\n    <link rel="stylesheet" href="../stylesheets/with-none.css" />\n    <script src="../scripts/with-none.js"><\/script>\n  </head>\n  <body>\n    Hello world\n  </body>\n</html>\n';

  // test/input/scripts/js-import-html.js
  console.log("html:", dead_end_default);
})();
