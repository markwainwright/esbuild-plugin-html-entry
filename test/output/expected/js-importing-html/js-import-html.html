<!doctype html>
<html>
  <head>
    <script src="js-import-html.js" integrity="sha256-hHTelIxHWrmWUOPhxwBuwtyY7p/af8sZRW3yzKFwTSM="></script>
  </head>
  <body>
    Hello world
  </body>
</html>
